samples/dictionary.sample.yml

```diff
@@ -0,0 +1,763 @@
+  FST:
+    lemma: wâsênamawêw
+    stem:  wâsênamaw-
+  head:     wâsênamawêw
+  paradigm: VTA
+  POS:      V
+  key:      wâsênamawêw
+  senses:
+    - definition: s/he puts the lights on for s.o.
+      sources:    [CW]
+    - definition: s/he lights (it/him) for s.o.
+      sources:    [CW]
+
+- category: VTA-3
+  FST:
+    lemma: mâciswêw
+    stem:  mâcisw-
+  head:     mâciswêw
+  paradigm: VTA
+  POS:      V
+  key:      mâciswêw
+  senses:
+    - definition: s/he cuts s.o. open
+      sources:    [CW]
+
+- category: VTA-4
+  FST:
+    lemma: kispêwâtêw
+    stem:  kispêwât-
+  head:     kispêwâtêw
+  paradigm: VTA
+  POS:      V
+  key:      kispêwâtêw
+  senses:
+    - definition: s/he defends s.o., s/he is overprotective of s.o.
+      sources:    [CW, MD]
+
+- category: VTA-5
+  FST:
+    lemma: ay-itêw
+    stem:  ay-it-
+  head:     ay-itêw
+  paradigm: VTA
+  POS:      V
+  key:      ay-itêw
+  senses:
+    - definition: s/he says so to s.o, s/he says so of s.o.
+      sources:    [CW]
+
+- category: VTI-1
+  FST:
+    lemma: kâhtinam
+    stem:  kâhtin-
+  head:     kâhtinam
+  paradigm: VTI
+  POS:      V
+  key:      kâhtinam
+  senses:
+    - definition: s/he obtains s.t., s/he attains s.t., s/he procures s.t.
+      sources:    [CW]
+    - definition: s/he pushes s.t.
+      sources:    [CW]
+
+- category: VTI-2
+  FST:
+    lemma: nahêkâcihtâw
+    stem:  nahêkâcihtâ-
+  head:     nahêkâcihtâw
+  paradigm: VTI
+  POS:      V
+  key:      nahêkâcihtâw
+  senses:
+    - definition: s/he is sparing of s.t.
+      sources:    [CW]
+
+- category: VTI-3
+  FST:
+    lemma: oscoscocasiw
+    stem:  oscoscocasi-
+  head:     oscoscocasiw
+  paradigm: VTI
+  POS:      V
+  key:      oscoscocasiw
+  senses:
+    - definition: s/he coughs a little
+      sources:    [CW]
+    - definition: s/he coughs up a little of something
+      sources:    [CW]
+
+- category: NA-3
+  FST:
+    lemma: atim
+    stem:  atimw-
+  head:     atim
+  paradigm: NA
+  POS:      N
+  key:      atim
+  senses:
+    - definition: dog
+      sources:    [CW, MD]
+    - definition: horse
+      sources:    [CW]
+    - definition: beast of burden
+      sources:    [CW]
+  forms:
+    - transcription: têm
+    - transcription: atimw
+
+- category: NA-3
+  FST:
+    lemma: atâhk
+    stem:  atâhkw-
+  head:     atâhk
+  paradigm: NA
+  POS:      N
+  key:      atâhk
+  senses:
+    - definition: star
+      sources:    [CW, MD]
+
+- category: NA-1
+  FST:
+    lemma: acâhkos
+    stem:  acâhkos-
+  head:     acâhkos
+  paradigm: NA
+  POS:      N
+  key:      acâhkos
+  senses:
+    - definition: star, little star
+      sources:    [CW, MD]
+
+- category: VAI-1
+  FST:
+    lemma: nîmiw
+    stem:  nîmi-
+  head:     nîmiw
+  paradigm: VAI
+  POS:      V
+  key:      nîmiw
+  senses:
+    - definition: s/he dances
+      sources:    [CW]
+
+# nîminâniwan does not (and should not) have a LEXC entry
+- category: VAI-1
+  head:     nîminâniwan
+  paradigm: VAI
```

Comment:
How does iDict know to use nîmiw as the FST/lemma for generating the paradigm?

If we don't want to have iDict or the importation process figure this out, should be not here give the FST/lemma, resulting from the FST analysis? (And we might then need to disambiguate that with the entry key for the entry having as its head such an FST lemma).